packages/ember-metal/lib/computed.js

```diff
@@ -67,44 +66,69 @@ function UNDEFINED() { }
   tom.get('fullName') // 'Tom Dale'
   ```
 
-  You can also define what Ember should do when setting a computed property.
-  If you try to set a computed property, it will be invoked with the key and
-  value you want to set it to. You can also accept the previous value as the
-  third parameter.
+  You can also define what Ember should do when setting a computed property by providing additional function (`set`) in hash argument.
+  If you try to set a computed property, it will try to invoke setter accessor function with the key and
+  value you want to set it to as arguments.
 
   ```javascript
-  var Person = Ember.Object.extend({
+  let Person = Ember.Object.extend({
     // these will be supplied by `create`
     firstName: null,
     lastName: null,
 
-    fullName: function(key, value, oldValue) {
-      // getter
-      if (arguments.length === 1) {
-        var firstName = this.get('firstName');
-        var lastName = this.get('lastName');
+    fullName: Ember.computed('firstName', 'lastName', {
+      get(key) {
+        let firstName = this.get('firstName'),
+            lastName  = this.get('lastName');
 
         return firstName + ' ' + lastName;
+      },
+      set(key, value) {
+        let [firstName, lastName] = value.split(' ');
 
-      // setter
-      } else {
-        var name = value.split(' ');
-
-        this.set('firstName', name[0]);
-        this.set('lastName', name[1]);
+        this.set('firstName', firstName);
+        this.set('lastName', lastName);
 
         return value;
       }
-    }.property('firstName', 'lastName')
+    })
   });
 
-  var person = Person.create();
+  let person = Person.create();
 
   person.set('fullName', 'Peter Wagenet');
   person.get('firstName'); // 'Peter'
   person.get('lastName');  // 'Wagenet'
   ```
 
+  Be careful - if you try to set a computed property and it won't have setter accessor function defined you will overwrite computed property with normal property (no longer computed) that won't change if dependencies change.
```

Comment:
This smacks of "NOTE:" style writing, which we try to avoid. Can you phrase this in positive, affirmative language instead?